docs/analysis.md

```diff
@@ -419,7 +419,7 @@ B* b;
 b->f(); // Z
 ```
 
-We will generate generate a source record at the line marked Z with
+We will generate a source record at the line marked Z with
```

Comment:
This is a relatively new feature, but I think my manual corrections are correct...